src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SDSEarlyReleaseDefaultingRulesService.kt

```diff
@@ -98,6 +105,26 @@ class SDSEarlyReleaseDefaultingRulesService(
     }
   }
 
+  fun handleCRDEqualsEligibilityDateAndTrancheDate(dates: MutableMap<ReleaseDateType, LocalDate>, tranche: TrancheOne) {
+    if (dates.containsKey(ReleaseDateType.CRD) || dates.containsKey(ReleaseDateType.ARD)) {
+      val controllingDate = getControllingDate(dates)
+
+      val eligibilityDates = listOf(
+        ReleaseDateType.HDCED4PLUS,
+        ReleaseDateType.HDCED,
+        ReleaseDateType.ERSED,
+      )
+
+      eligibilityDates.forEach { dateType ->
+        if (dates[dateType] == controllingDate.second &&
+          (dates[dateType] == tranche.trancheCommencementDate || dates[dateType] == tranche.trancheTwoCommencementDate)
+        ) {
+          dates.remove(dateType)
+        }
+      }
```

Comment:
I'm sure Luke will be pleased that some of the breakdown changes won't ever be seen :)

Reply:
Shhhh